scss/bitstyles/ui/stats.stories.mdx

```diff
@@ -58,7 +58,7 @@ Highlighting a few statistics presents us the opportunity to give context and in
   <Story name="Stats (with change indicators)">
     {`
       <div class="a-content">
-        <dl class="u-grid u-grid-cols-3@m u-gap-m">
+        <div class="u-grid u-grid-cols-3@m u-gap-m">
```

Comment:
It is not possible to write a valid definition list that has this layout. There can't be a double-nested div in ` dl`, so it's not possible to put a badge in there. It is very limited which children are valid in a `dl`.

This is valid:
```html
<dl>
	<dt></dt>
	<dd></dd>
</dl>
```
```html
<dl>
	<div>
		<dt></dt>
		<dd></dd>
	</div>
</dl>
```

This is not valid
```html
<dl>
	<div>
		<div>
			<dt></dt>
			<dd></dd>
		</div>
	</div>
</dl>
```